Simplified/AppInfrastructure/NYPLAppDelegate.m

```diff
@@ -127,6 +127,7 @@ - (BOOL)application:(__unused UIApplication *)app
   
   NYPLBook *book = [NYPLBook bookWithEntry:entry];
   if (!book) {
+      // Localization???
```

Comment:
if I recall well, NYPLAlertUtils performs localization. It should be documented, but if not that's one thing to fix.

Reply:
It does yes